packages/katello/katello/katello.spec

```diff
@@ -194,6 +194,10 @@ Useful utilities for managing Katello services
 %{_sysconfdir}/bash_completion.d/katello-service
 
 %changelog
+* Sun Jul 21 2019 Jonathon Turel <jturel@gmail.com> 3.12.1-2
```

Comment:
Is that supposed to go out as a fix to 3.12.1? Or should this be part of 3.12.2?

If the latter, I'd just include/update this PR when I'll be doing the overall 3.12.2 packaging for you.

Reply:
It's for 3.12.2

Can you reword your suggestion - want to make sure I'm following

Reply:
Chris said there will be a 3.12.2 soon. I think there is no reason to release a 3.12.1-2 now just to be overwritten by 3.12.2-1 days later.

So I'd just fold this PR into the 3.12.2 one as soon that's open (and leave this open until then for documentation purposes).

Reply:
Makes sense! thx for the guidance